boost_1_71_0/boost_version.txt

```diff
@@ -0,0 +1 @@
+1.71.0
```

Comment:
This version information _is_ in `boost/version.hpp`.  But perhaps you think it less obscure to duplicate it in a plain text file as opposed to a code file.

Reply:
I didn't know it was in there already. I'll remove this file and merge into the other branch.